src/main/kotlin/edu/carroll/polyominoes/web/controller/account/LoginController.kt

```diff
@@ -0,0 +1,22 @@
+package edu.carroll.polyominoes.web.controller.account
+
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+
+
+@Controller
+@EnableWebSecurity
+class LoginController {
+
+    /**
+     * Returns the login page when an HTML get request for /login is received.
+     *
+     * @return "login"
+     */
+    @GetMapping("/login")
+    fun loginGet(): String {
+        return "login"
+    }
+
```

Comment:
This is missing the code to handle the POST from the login page, so the application doesn't run as provided to me.

Reply:
It's working now, not sure why it didn't work earlier today.